### ISSUE TITLE
Feature request: brms

### ISSUE THREAD

**Issue by klc108** [jasp-stats/jasp-issues#802]
Hi JASP Team,
Would it be possible to implement brms in JASP for Bayesian modeling within JASP? The No U-Turn Sampler (NUTS) for MCMC is extremely fast and efficient and the brms front end to the Stan language makes everything much easier for the  Bayesian analyst than using Stan or using JAGS. 

Thank you for recently adding JAGS. It would be great to add brms in the next JASP version.
Gracias,
klc108

**Comment by klc108** [comment]
Thank you, EJ. That's great news!

All the best,
Ken (klc108)

Kenneth L. Cavanaugh, Ph.D.
Research Professor
Institute of Science, Technology & Public Policy
Maharishi International University MR 774
1000 North 4th Street
Fairfield, IA 52557
office: 641-209-5510

**Comment by nahorp** [comment]
2nd

Also, Bayes Factors for Bayesian Mixed Models?

**Comment by EJWagenmakers** [comment]
Yes, clearly on our agenda. The BayesFactor package already implements a mixed model, but we want to look at this a little more closely.
E.J.

**Comment by tomtomme** [comment]
@klc108 & @nahorp 
Is brms now implemented / used via bayesian regression, mixed models etc.? 
I cannot directly spot it in the docs or the GUI and am not very familiar with all the bayesian lingo yet.

**Comment by TarandeepKang** [comment]
@tomtomme as far as I know, brms is not available yet. It is essentially an extremely flexible way to fit a huge variety of Bayesian regression models. @EJWagenmakers is this still on the agenda? I definitely hope so!

Best,

Tarandeeep

**Comment by EJWagenmakers** [comment]
Yes it is a good idea, still very much on the agenda
